controlplane/pkg/nsmd/nsmd_crossconnect_client_test.go

```diff
@@ -0,0 +1,63 @@
+package nsmd
+
+import (
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/registry"
+	model_ "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/model"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/monitor/crossconnect_monitor"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/monitor/remote_connection_monitor"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/services"
+	. "github.com/onsi/gomega"
+	"testing"
+	"time"
+)
+
+func TestMonitorEndpoints(t *testing.T) {
+	RegisterTestingT(t)
+
+	model := model_.NewModel()
+
+	endpointStateManager := NewEndpointStateManagerStub(model)
+	model.AddListener(endpointStateManager)
+
+	client := NewMonitorCrossConnectClient(mockCrossConnectMonitor(), mockConnectionMonitor(), mockXconManager(),
+		endpointStateManager)
+	model.AddListener(client)
+
+	model.AddEndpoint(createEndpoint("ws1", "nse1"))
+	model.AddEndpoint(createEndpoint("ws2", "nse2"))
+	model.AddEndpoint(createEndpoint("ws3", "nse3"))
+
+	endpointStateManager.KillEndpoint("ws1", "nse1")
+
+	_ = model.DeleteEndpoint("nse2")
```

Comment:
Same, unnecessary assigning

Reply:
It is for suppressing GoLand 'Unhandled error' check